scripts/fix-quotes.sh

```
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Script to fix quote style issues in JavaScript files
+cd "$(dirname "$0")/.."
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)